test/BUILD.gn

```diff
@@ -451,13 +459,19 @@ test("brave_unit_tests") {
 
   if (enable_brave_perf_predictor) {
     sources += [
-      "//brave/components/brave_perf_predictor/browser/named_third_party_registry_unittest.cc",
       "//brave/components/brave_perf_predictor/browser/bandwidth_linreg_unittest.cc",
       "//brave/components/brave_perf_predictor/browser/bandwidth_savings_predictor_unittest.cc",
       "//brave/components/brave_perf_predictor/browser/p3a_bandwidth_savings_permanent_state_unittest.cc",
       "//brave/components/brave_perf_predictor/browser/p3a_bandwidth_savings_tracker_unittest.cc",
     ]
 
+    if (!is_android) {
+      sources += [
+        # Fails to load `brave/components/brave_perf_predictor/resources/entities-httparchive-nostats.json` file
```

Comment:
what's failing here? likely the loading needs fixing, but the feature wouldn't work if this actually fails to load

Reply:
We just don't have such file on android. Should it be there?

Reply:
we should either disable `brave_perf_predictor` completely on Android, or have the file, yes